neptune_tensorboard/sync/tensorflow_data_sync.py

```diff
@@ -49,8 +49,15 @@ def run(self):
                         pass
 
     def _load_single_run(self, path):
-        click.echo("Loading {}...".format(path))
         run_path = os.path.relpath(path, self._path)
+
+        # Ignore files generated by Tensorboard plugins as
+        # those do not correspond to Tensorlfow runs
+        # and as such can be considered rubbish.
+        if run_path.startswith('train/plugins') or run_path.endswith('profile-empty'):
```

Comment:
This solves the particular problem that was reproduced, but a better fix would be to check file whether it contains non-empty experiment run data. But we'd need someone on the team with better understanding of TF internals.